packages/flutter/lib/src/cupertino/switch.dart

```diff
@@ -189,6 +192,7 @@ const double _kTrackInnerEnd = _kTrackWidth - _kTrackInnerStart;
 const double _kTrackInnerLength = _kTrackInnerEnd - _kTrackInnerStart;
 const double _kSwitchWidth = 59.0;
 const double _kSwitchHeight = 39.0;
+const double _kCupertinoSwitchDisabledOpacity = 0.5;//value appears to be accurate in comparison to the actual iOS switch
```

Comment:
This comment is too long for the line. Would you mind moving the comment up above the constant declaration line and form it as a sentence? Also, would you mind pointing out that you eye-balled the value rather than used any particular tool? E.g.,

```dart
// Opacity of a disabled switch, as eye-balled from iOS.
const double _k....
```

Reply:
Made the change